test/user.test.js

```diff
@@ -214,6 +214,18 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('should invalidate accessToken once user is deleted', function(done) {
```

Comment:
`it('should invalidate the user\'s access token when the user is deleted', function(done) {`